### ISSUE TITLE
Offical way to add sponser chanel

### ISSUE THREAD

**Issue by Redshado0w** [FreedomPrevails/JSMTProxy#12]
hi all dears
today telegram created mtpro bot for server managers
just send your proxy to this bot and add your chanel
sorry for bad english
@MTProxybot

**Comment by p1ratrulezzz** [comment]
Yes, but it's not enough. On proxy server side need to send proxy tag provided by this bot. I hope it will be added soon to all implementations of mtproxy

**Comment by Redshado0w** [comment]
we will hope you add this

**Comment by WebShark025** [comment]
duplicate #7

**Comment by ErfanDevelop3r** [comment]
Hello Guys.
I found One Official Way To Add Your Sponser Channel. 
1. Go To this Bot: @MTProxybot
2. Click On Start Button
3. Send /newporoxy Command
4. Add Your Proxy Information
5. Send /manageproxy Command
6. Choose Your Server
7. Click On Set Permitions Button
8. Send Your Channel Id
9. Finished. Enjoy It From This Tutorial 😘❤️

**Comment by MTProto** [comment]
Take a look at [MTProtoProxy-npm](https://www.npmjs.com/package/mtprotoproxy) or [MTProtoProxy-github](https://github.com/MTProto/MTProtoProxy). Using the npm package you can add sponsor channel.
